.gitattributes

```
@@ -11,3 +11,14 @@
 *.pcm binary
 
 Dockerfile.dev linguist-language=Dockerfile
+
+# Generated files
+CODEOWNERS linguist-generated=true
+Dockerfile linguist-generated=true
+homeassistant/generated/*.py linguist-generated=true
```

```suggestion
homeassistant/generated/* linguist-generated=true
```
Any particular reason this was only marked for python files, as json files are also generated.

---

No reason, just didn't notice a json there. Feel free to add it as well. Btw, `__init__.py` is the only non-generated file there, ideally it needs to be excluded but I don't expect it to change.